css/types/image.json

```diff
@@ -405,6 +406,559 @@
+                    ]
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "49"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "44",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "layout.css.prefixes.webkit",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
+                "firefox_android": [
+                  {
+                    "version_added": "16",
+                    "notes": "Before Firefox 36, gradients weren't applied on the pre-multiplied color space, leading to shades of grey unexpectedly appearing when used with transparency."
+                  },
+                  {
+                    "prefix": "-moz-",
+                    "version_added": "4",
+                    "notes": [
+                      "Since Firefox 42, the prefixed version of gradients can be disabled by setting <code>layout.css.prefixes.gradients</code> to <code>false</code>."
+                    ]
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "49"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "44",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "layout.css.prefixes.webkit",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
+                "ie": {
+                  "version_added": "10",
+                  "notes": "Internet Explorer 5.5 through 9.0 supported gradients via a proprietary filter: <code><a href='https://developer.mozilla.org/docs/Web/CSS/-ms-filter#Gradient'>-ms-filter: progid:DXImageTransform.Microsoft.Gradient()</a></code>."
+                },
+                "opera": [
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
+                    "version_added": "11.6",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
+                    "version_added": "15"
+                  }
+                ],
+                "opera_android": [
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
+                    "version_added": "12",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
+                    "version_added": "15"
```

Comment:
The first Opera for Android release which used Chromium was 14, not 15, which was the first desktop Opera release which used Chromium.

See #1712